contracts/StablecoinConverter.sol

```diff
@@ -219,11 +216,17 @@ contract StablecoinConverter is EpochTokenLocker {
         uint128 executedSellAmount,
         uint128 buyTokenPrice,
         uint128 sellTokenPrice
-    ) internal pure returns (uint128) {
-        return uint128(
+    ) internal view returns (uint128) {
+        uint128 buyAmount = uint128(
             executedSellAmount.mul(buyTokenPrice) /
             sellTokenPrice
         );
+        // executedBuyAmount = buyAmount * (1 - (1/feeDenominator)
+        //           = buyAmount - buyAmount/feeDenominator (*)
+        //           = (buyAmount * feeDenominator)/ feeDenominator - buyAmount/feeDenominator
+        //           = (buyAmount * feeDenominator - buyAmount) / feeDenominator
+        //           = (buyAmount* (feeDenominator - 1)/feeDenominator
```

Comment:
Should these commented lines be removed.

Reply:
Felix asked for them :) 
I think anything that helps readability is a plus